README.md

```diff
@@ -42,6 +42,9 @@ These environment variables are supported:
 | DB_NAMES             |               | Names of the databases for which a backup should be created.                                                                                                                                                                     |
 | TZ                   |               | Timezone to use.                                                                                                                                                                                                                 |
 | CRON_SCHEDULE        | `0 1 * * sun` | Cron schedule for running backups. NOTE: There is no check if there's already a backup running when starting the backup job. Therefore time interval needs to be longer than the maximum expected backup time for all databases. |
+| BACKUP_CLEANUP      | `false` | Set to "true" if you want to let the cronjob remove files older than $BACKUP_AGE days |
+| BACKUP_AGE          | `7` | Number of days to keep backups in backup directory |
+| BACKUP_DIR          | `/backup` | Backup directory in the MSSQL container  |
```

Comment:
Why do we need the possibility to specify the backup directory BACKUP_DIR?

Reply:
Flexibility?
By making it a variable you can set a specific path without rebuilding your image.
As you like, it's just a suggestion.

Reply:
I would remove them. I see no use for it at the moment. For the volume mapping it is easier we have `/backup` fix defined folders inside the containers.

Reply:
So you would remove that variable and want me to edit the PR? ;-)

Reply:
Sorry I was unclear. It would be nice if you could do that in your PR. I will test your solution as soon as possible.

Reply:
done already a few hours ago in : https://github.com/bbtsoftware/docker-mssql-backup/pull/17/commits/cca6df533340d2e77873a580bc5717a71950d5b6

I am on vacation from tomorrow.

Reply:
Thank you very much for your contribution. 👍 
I will finish and test your pr. Enjoy your vacation in switzerland. :-)